_includes/configuration.md

```diff
@@ -90,32 +86,18 @@ necessary to alter the `max_connections` setting.
 
 4.  Select **Configure Application** from the menu.
 
-5.  You are prompted to create or fetch a security key.
```

Comment:
Why is this removed?

Reply:
It is removed as per @Fryguy 's comments in https://github.com/ManageIQ/manageiq-documentation/pull/1626#